SpatialGDK/Source/SpatialGDKEditor/Private/SpatialLaunchConfigCustomization.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialLaunchConfigCustomization.h"
+
+#include "SpatialGDKSettings.h"
+#include "SpatialGDKEditorSettings.h"
+
+#include "PropertyCustomizationHelpers.h"
+#include "PropertyHandle.h"
+#include "Widgets/SToolTip.h"
+#include "IDetailChildrenBuilder.h"
+#include "Widgets/Text/STextBlock.h"
+#include "IDetailGroup.h"
+
+TSharedRef<IPropertyTypeCustomization> FSpatialLaunchConfigCustomization::MakeInstance()
+{
+	return MakeShared<FSpatialLaunchConfigCustomization>();
+}
+
+void FSpatialLaunchConfigCustomization::CustomizeHeader(TSharedRef<class IPropertyHandle> StructPropertyHandle, class FDetailWidgetRow& HeaderRow, IPropertyTypeCustomizationUtils& StructCustomizationUtils)
+{
+	
+}
```

Comment:
this is a virtual method than can be completely omitted from definition / declaration in this class?

Reply:
Pure virtual.

Reply:
I don't understand what you mean? This function doesn't have the pure virtual macro in the header file? Even if it did we could remove the definition, and since it doesn't, can we still remove both definition and declaration?

Reply:
It's declared as a pure virtual function in the interface definition, so it has to be defined in any class that inherits it.

Reply:
ah yea ofc, my bad